modules/gridmap/doc_classes/GridMap.xml

```diff
@@ -211,6 +214,8 @@
 		<member name="collision_mask" type="int" setter="set_collision_mask" getter="get_collision_mask" default="1">
 			The physics layers this GridMap detects collisions in. See [url=https://docs.godotengine.org/en/latest/tutorials/physics/physics_introduction.html#collision-layers-and-masks]Collision layers and masks[/url] in the documentation for more information.
 		</member>
+		<member name="layers" type="int" setter="set_navigation_layers" getter="get_navigation_layers" default="1">
```

Comment:
Ah yeah, you are right. I'll rename the variable.
I don't know, I'd put an "s" because every bit of the bitmask is corresponds to a layer. So you can assign several layers to a same gridmap. I'd say the plural form makes more sense here.

Reply:
Yeah I think so too. We should discuss with the physics folks whether to rename `collision_layer` everywhere to `collision_layers`.